### PR TITLE
cps-datensysteme.de.xml

### PR DESCRIPTION
**Pull request by zneaa**
https://github.com/EFForg/https-everywhere/issues/2324

### CODE REVIEW

File: src/chrome/content/rules/cps-datensysteme.de.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="cps-datensysteme.de">
+    <target host="cps-datensysteme.de"/>
+    <target host="*.cps-datensysteme.de" />
+    
+    <test url="http://blog.cps-datensysteme.de/" />
+    <test url="http://www.cps-datensysteme.de/" />
+    <test url="http://go.cps-datensysteme.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/cps-datensysteme.de.xml]: `news.cps-datensysteme.de`

File: src/chrome/content/rules/cps-datensysteme.de.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="cps-datensysteme.de">
+    <target host="cps-datensysteme.de"/>
+    <target host="*.cps-datensysteme.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/cps-datensysteme.de.xml]: Please avoid wildcards if possible and list all targets instead.

**Comment by jeremyn** [comment]
@zneaa Are you still interested in working on this pull request?

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @zneaa .

**Pull request closed without merging**